src/pages/ActiveTasks/ActiveTasks.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState, ChangeEvent } from "react";
+import ActiveTaskTile from "../../components/ActiveTaskTile/ActiveTaskTile";
+import Navigation from "../../components/Navigation/Navigation";
+import tasks from "../../MockData/tasks";
+import "./ActiveTasks.scss";
+import Header from "../../components/Header/Header";
+import TextField from "@mui/material/TextField";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
We should be able to use the search icon from mui instead of Font Awesome. It makes more sense as we are using them in the rest of the project. 

- https://mui.com/material-ui/material-icons/?query=search
- https://mui.com/material-ui/icons/